public/app/features/teams/TeamMemberRow.tsx

```diff
@@ -106,4 +104,5 @@ const mapDispatchToProps = {
   updateTeamMember,
 };
 
-export default connect(mapStateToProps, mapDispatchToProps)(TeamMemberRow);
+const connector = connect(mapStateToProps, mapDispatchToProps);
```

Comment:
```suggestion
const connector = connect(null, mapDispatchToProps);
```
Can just use `null` instead of defining empty `mapStateToProps`.